.github/workflows/docker.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker
+
+on:
+  pull_request:
+    paths:
+    - 'Dockerfile'
+    - '.github/workflows/docker.yml'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 40
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build container image
+      run: docker build -t test-container-image .
+    - name: Launch container
+      run: docker run -dp 3000:3000 test-container-image
+      timeout-minutes: 5
+    - run: docker ps
+    - name: Wait for Metabase to start
+      run: sleep 25
```

Comment:
Are you sure Metabase will always start in 25 seconds? Maybe it's better to do something like what we do in CircleCI and wait for the port to start accepting incoming connections

Reply:
Good point. Somehow I couldn't make the `nc` check work (not sure why) but let me see if a loop with `curl` will do the trick.

Reply:
I've tweaked it so it doesn't wait for a hardcoded period of time anytime. Please take a look!